### PR TITLE
DRAFT Compiler FE: Enable SegmentSum operator 

### PR DESCRIPTION
**Pull request by d-krylov**
Draft for SegmentSum operator 

ONE-DCO-1.0-Signed-off-by: Denis Krylov <krylovdenism@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
There are still unfinished operators in #1167.
What are you going to do with them?
@binarman , what is going on?

**Comment by seanshpark** [comment]
@binarman , as like previous tasks to enable operators, please do not start (4) #1606 before ending (3)

**Comment by binarman** [comment]
@seanshpark 
This is misunderstanding of what is left in #1167 
You are right, we will finish left tasks.

**Comment by d-krylov** [comment]
@seanshpark, @binarman, PTAL, please.

File: compiler/tflchef/tflite/src/Op/SegmentSum.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  assert(tensor->type() == tflite::TensorType::TensorType_INT32);
+  const tflite::Buffer *buffer = import->buffers()->Get(tensor->buffer());
+
+  if (buffer && buffer->data())
+  {
+    auto vec = extract_buffer<int32_t>(buffer);
+    import->set_tensor_filler(inputs[1], vec);
+  }
+}
+tflchef::Operation *TFliteOpSegmentSum::build(const tflite::Operator *op, TFliteImport *import,
```
Comment (seanshpark) [inline compiler/tflchef/tflite/src/Op/SegmentSum.cpp:39]: please add an empty space before here

File: compiler/tflchef/tflite/src/Op/SegmentSum.cpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  const tflite::Tensor *tensor = import->tensors()->Get(inputs[1]);
+  assert(tensor->type() == tflite::TensorType::TensorType_INT32);
+  const tflite::Buffer *buffer = import->buffers()->Get(tensor->buffer());
+
+  if (buffer && buffer->data())
+  {
+    auto vec = extract_buffer<int32_t>(buffer);
+    import->set_tensor_filler(inputs[1], vec);
+  }
```
Comment (seanshpark) [inline compiler/tflchef/tflite/src/Op/SegmentSum.cpp:37]: If possible, please use `fill_tensor_to_import()`. You can refer to `Add` Op.

**Comment by seanshpark** [comment]
Looks OK with minor commnets. Thank you! Please proceed with posting PRs

**Comment by binarman** [comment]
Everything is merged, closing

**Pull request closed without merging**